src/eShop.ServiceDefaults/OpenApi.Extensions.cs

```diff
@@ -64,6 +67,13 @@ public static IHostApplicationBuilder AddDefaultOpenApi(
                     options.ApplyAuthorizationChecks([.. scopes.Keys]);
                     options.ApplySecuritySchemeDefinitions();
                     options.ApplyOperationDeprecatedStatus();
+                    // Clear out the default servers so we can fallback to
+                    // whatever ports have been allocated for the service by Aspire
```

Comment:
What fallback exactly? I assume `Servers` won't be populated by anyone else?

Reply:
It would be up to the consumer of the OpenAPI document to set it. When you're in Scalar, it will use the host that the Scalar UI is served on as the base host name. If you import the OpenAPI document into a tool like Bruno or Postman, they have UIs where you can set the base host name. This is a result of this issue: https://github.com/dotnet/aspnetcore/issues/57332